samples/js/picasa/slurp/src/oauth.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+import {createServer as createHttpServer} from 'http';
+import {OAuth2} from 'oauth';
```

Comment:
Anything which needs refresh tokens yes, but nothing else does yet. We can cross that bridge.